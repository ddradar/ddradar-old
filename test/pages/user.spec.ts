import { createLocalVue, shallowMount, Wrapper } from '@vue/test-utils'
import Buefy from 'buefy'

import UserPage from '@/pages/user.vue'

jest.mock('~/plugins/firebase', () => {
  return {
    firestore: jest.fn()
  }
})
const localVue = createLocalVue()
localVue.use(Buefy)

describe('/user/', () => {
  let wrapper: Wrapper<UserPage>
  let vm: any
  beforeEach(() => {
    wrapper = shallowMount(UserPage, { localVue })
    vm = wrapper.vm
  })
  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })
  test('user is null default', () => {
    expect(vm.user).toBeNull()
  })
  describe.skip('asyncData Method', () => {})
  describe.skip('updateUserInfo Method', () => {})
})
