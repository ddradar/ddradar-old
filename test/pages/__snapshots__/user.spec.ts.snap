// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/user/ renders correcly 1`] = `<!---->`;
